tools/python3/bench.py

```
@@ -0,0 +1,271 @@
+            particles = int(bit[len(needle) :])
+    bits = bits[1][1:-1].split(",")
+    patches, cells, p_ghosts_cells, l_ghosts_cells = [int(s) for s in bits]
+    ratio = (p_ghosts_cells + l_ghosts_cells) / cells
+    stats = LevelStats(
+        ilvl, patches, cells, p_ghosts_cells, l_ghosts_cells, particles, ratio
+    )
+    # print("LevelStats", stats)
+    return stats
+
+
+def parse_summary_file(input):
```

Ruff (0.15.15)

[error] Function argument `input` is shadowing a Python builtin

(A002)